perfkitbenchmarker/linux_benchmarks/ping_benchmark.py

```diff
@@ -62,17 +62,41 @@ def Run(benchmark_spec):
     A list of sample.Sample objects.
   """
   vms = benchmark_spec.vms
-  if not vms[0].IsReachable(vms[1]):
-    logging.warn('%s is not reachable from %s', vms[1], vms[0])
+  results = []
+  for origin_index in [0, 1]:
```

Comment:
Addendum: the brackets are unnecessary and generate a new list object. They can be omitted:
```
for sending_vm, receiving_vm in vms, reversed(vms):
```